client.go

```diff
@@ -109,7 +109,7 @@ func (c *Client) newRequest(method string, action string, params Params, body in
 	))
 	req.Header.Set("Authorization", fmt.Sprintf("Basic %s", c.apiKey))
 	req.Header.Set("Accept", "application/xml")
-	req.Header.Set("X-Api-Version", "2.5")
+	req.Header.Set("X-Api-Version", "2.11")
```

Comment:
Do you know if there are any other any other API changes between 2.5 and 2.11 that could be breaking?

Reply:
I do not know of any breaking changes. I don't see any notes about breaking changes in the release notes from 2.5 to 2.12.